tests/test_evaluation/test_evaluators.py

```diff
@@ -347,22 +358,24 @@ def process_scores_(
             self.counter -= 1
 
     def finalize(self) -> MetricResults:  # noqa: D102
-        return RankBasedMetricResults.from_dict(
```

Comment:
was this change necessary? I made that funny class method to avoid diff 🙃

Reply:
It was only used once, and did not seem to serve a purpose anymore.

Reply:
Good enough!